models/commons/src/price/price-draft/builder.spec.ts

```diff
@@ -60,10 +56,10 @@ describe('builder', () => {
       expect.objectContaining({
         key: null,
         value: expect.objectContaining({
-          centAmount: expect.any(Number),
-          currencyCode: expect.any(String),
-          type: 'centPrecision',
-          fractionDigits: expect.any(Number),
+          centPrecision: {
+            centAmount: expect.any(Number),
+            currencyCode: expect.any(String),
+          },
```

Comment:
The graphql draft expects this signature:
![image](https://github.com/commercetools/test-data/assets/15024562/b3413cfb-0607-4f9c-8417-74745079bbb1)
and
![image](https://github.com/commercetools/test-data/assets/15024562/5a463740-b161-4510-95bb-3c8e4e1aa83d)

Rather than make _yet another model_ for a graphql-only model (BaseMoneyInput), I felt it fine to use `centPrecision: { ... }` in the transformer for `PriceDraft`